### PR TITLE
Convert to ES6 and add random color

### PR DESCRIPTION
**Pull request by charpeni**
- All components are updated to ES6 style.
- Updated instruction on README `gb-native-router` instead of `react-native-router`.
- Adding a feature that applies a random background color. This allows to clearly see the changes page and backstack.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,12 +166,12 @@ Events emitted by the router:
 A more advanced example: Twitter app
 ------------------------------------
 
-To see more of the router in action, you can check out the Twitter example app that comes with the package. Just make sure that you first drag all the images from ```node_modules/react-native-router/twitter-example/images``` to your project's Images.xcassets
+To see more of the router in action, you can check out the Twitter example app that comes with the package. Just make sure that you first drag all the images from ```node_modules/ga-native-router/twitter-example/images``` to your project's Images.xcassets
```
Comment (davidLeonardi) [inline README.md]: typo! should be "gb-native-router" not "ga-native-router"

**Comment by davidLeonardi** [comment]
Great job!

File: README.md (hunk 1 above)
Comment (charpeni) [inline README.md]: Thanks! It's fixed.

**Comment by ericraio** [comment]
this repo was created to be maintained, please merge! :)

**Comment by charpeni** [comment]
@ericraio Exactly.

@MikaelCarpenter just add someone in collaborators if you do not have time. :)

**Comment by davidLeonardi** [comment]
+1

**Comment by charpeni** [comment]
In a few days, if we didn't have news of @MikaelCarpenter, I will fork this package, update it and deploy it on npm.

**Comment by davidLeonardi** [comment]
This is insanity. But i agree in principle.
May I suggest that we give it a better name?
Any suggestions?
Can we please nominate more than one maintainer? (I'm volounteering here!)

**Comment by charpeni** [comment]
Sure, we should find a better name, unfortunately I have no idea.

> Can we please nominate more than one maintainer? (I'm volounteering here!)

Certainly, this package must evolve.

**Comment by johannhof** [comment]
Why not make an organization instead of forking it again and again. The license is MIT, so we could just copy the code into a new repository. This would also give @t4t5 and @MikaelCarpenter the chance to continue as maintainers if they want to.

**Comment by johannhof** [comment]
I camped https://github.com/react-native-router just to be sure. Feel free to do whatever you like with it.

**Comment by davidLeonardi** [comment]
@charpeni, @johannhof, @ericraio, please see #58

**Pull request closed without merging**